util/pastPerformance/testReleasesPerformance

```diff
@@ -95,9 +103,9 @@ export CHPL_TEST_PERF_DATE="03/29/16"
 export CHPL_HOME=$chpl_release_home/chapel-1.13.0
 testReleasePerformance
 
-export -n CHPL_TEST_PERF_DATE
+unset CHPL_TEST_PERF_DATE
```

Comment:
With export -n, if you said $CHPL_TEST_PERF_DATE later in this script, it will print whatever last thing it was set to. Whereas scripts invoked from it will not see it. This can easily be confusing IMO.